ltsp/server/ipxe/ltsp.ipxe

```diff
@@ -41,23 +40,19 @@ item --key x exit                 Exit iPXE and continue BIOS boot
 choose --timeout ${menu-timeout} --default ${img} img || goto cancel
 goto ${img}
 
-:images
-# The "images" method can boot anything in /srv/ltsp/images
-set cmdline_method root=/dev/nfs nfsroot=${srv}:/srv/ltsp ltsp.image=images/${img}.img loop.max_part=9
-goto ltsp
+# Jump targets setting correct ${img} var for menu items.
+:gotos
 
-:roots
-# The "roots" method can boot all /srv/ltsp/roots
-set cmdline_method root=/dev/nfs nfsroot=${srv}:/srv/ltsp/${img}
-goto ltsp
+# Jump targets with the respective boot_method.
+:boot_methods
 
 :ltsp
-# :images and :roots jump here after setting cmdline_method
-set cmdline ${cmdline_method} ${cmdline_ltsp} ${cmdline_client}
+# :images and :roots jump here after setting cmdline_boot_method
+set cmdline ${cmdline_boot_method} ${cmdline_ltsp} ${cmdline_client}
 # In EFI mode, iPXE requires initrds to be specified in the cmdline
-kernel /ltsp/${img}/vmlinuz initrd=ltsp.img initrd=initrd.img ${cmdline}
-initrd /ltsp/ltsp.img
+kernel /ltsp/${img}/vmlinuz initrd=initrd.img initrd=ltsp.img ${cmdline}
 initrd /ltsp/${img}/initrd.img
+initrd /ltsp/ltsp.img
```

Comment:
ltsp.img needs to be loaded first; see https://github.com/ltsp/ltsp/issues/9

Reply:
Alright, my bad.

Reply:
`kernel /ltsp/${img}/vmlinuz`
Should actually change too.
A subdir for the boot_method should be prepended in case one has boot targets with the same name (e.g. ubuntu) in different dirs for different boot methods.

Reply:
Bugfix pushed